INSTALL.md

```diff
@@ -20,7 +20,7 @@ Setup a virtual environment in conda named `qiita` by executing the following:
 
 ```bash
 conda config --add channels conda-forge
-conda create -q --yes -n qiita python=3.9 pip libgfortran numpy nginx cython redis
+conda create -q --yes -n qiita python=3.9 pip libgfortran numpy nginx
```

Comment:
I would include cython and redis packages as well, just like we do in the GitHub workflows configuration file.

Reply:
Note that these were not added.